km_api/know_me/factories.py

```diff
@@ -121,6 +121,19 @@ class Meta:
         model = models.Profile
 
 
+class ProfileAccessorFactory(factory.django.DjangoModelFactory):
+    """
+    Factory for generating ``ProfileAccessor`` instances.
+    """
+    km_user_accessor = factory.SubFactory(
+        'know_me.factories.KMUserAccessorFactory')
+    profile = factory.SubFactory('know_me.factories.ProfileFactory')
+    can_write = False
```

Comment:
There's no need to specify `can_write = False` since that field has a default defined on the model itself.